src/components/details-page/components/details-page/styled.ts

```diff
@@ -220,6 +220,8 @@ const Content = styled.main`
   flex-direction: column;
   gap: ${t.spacing('S12')};
   z-index: 10;
+  max-width: 74%;
```

Comment:
Jeg hadde gått for å fjerne "white-space: nowrap;" på distribution summary title (h4)

Reply:
Bra observasjon! Jeg måtte endre flex-direction i Summary componenten for at teksten og nedlastings-ikonene ikke skulle se rar ut. Håper det ikke får konsekvenser andre komponenter som bruker Summary.